### PR TITLE
Search schema filter

### PR DESCRIPTION
**Pull request by wpf500**
Initial work on the search filters. Added some simple CSS and implemented the basic schema filter.

- Renamed `Search` to `SearchFilter` for clarity

image

## TODO
- Keep count of other schemas when filtered (backend API change required)
- Fix some styles (e.g. font colors) and fix Blueprint bug where the selected tab highlight doesn't update if tab text changes
- Collections list and filter
- List view

### CODE REVIEW

File: src/components/SearchFilter.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        [param]: newValue
+      }
+    });
+  }
+
+  onTextChange(e) {
+    this.handleParamChange('q', e.target.value);
+  }
+
+  onEntityChange(type) {
```
Comment (pudo) [inline src/components/SearchFilter.js]: Entities are real things in Aleph, and this doesn't touch them. I would call it ``onSchemaChange``?

File: src/components/SearchFilter.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@import "~@blueprintjs/core/dist/variables";
+
```
Comment (pudo) [inline src/components/SearchFilter.scss:2]: Ok, this seems fair in terms of CSS. I like that you're using the $pt vars. Are you sure about the mixed _ and -- convention for CSS class names?

File: src/components/SearchFilterEntities.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { FormattedMessage, FormattedNumber } from 'react-intl';
+import { Tab2, Tabs2 } from '@blueprintjs/core';
+
```
Comment (pudo) [inline src/components/SearchFilterEntities.js]: Ok, so all of this is about Schema, not entities (Schema applies to both entities and Documents)

**Review by pudo — CHANGES_REQUESTED** [review]
Looking good :)

File: src/components/SearchFilter.scss (hunk 2 above)
Comment (wpf500) [reply src/components/SearchFilter.scss:2]: http://getbem.com/introduction/ as discussed on Slack

**Review by pudo — APPROVED** [review]

**Pull request merged**